tests/python/test-specs.yml

```diff
@@ -0,0 +1,15 @@
+description: End-to-end tests for python initcontainer
+custom_test_key: tags.testKey
+scenarios:
+  - description: This scenario will verify that a transaction is reported by the test app after a curl request
+    before:
+      - helm install test-python ./chart/ --set=scenarioTag="${SCENARIO_TAG}" -n default
+      - sleep 5
+      - kubectl wait --for=condition=Ready -n default --all pods
+      - curl --fail-with-body $(minikube service test-app-python-service --url -n default)
+    tests:
+      nrqls:
+        - query: SELECT latest(duration) AS duration FROM Transaction WHERE appName = 'k8s-e2e-test-app-python'
```

Comment:
The tests fail if no transaction is generated, which should be what we want since we want a full E2E test. There should always be a transaction reported or we've got issues.